### PR TITLE
Separate backlinks-feature from show-arrows-in-ui feature

### PR DESCRIPTION
**Pull request by goliath-walker**
As suggested in https://github.com/treymo/joplin-link-graph/pull/50#issuecomment-1040423617

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ interface GraphData {
   currentNoteID: string;
   nodeFontSize: number;
   nodeDistanceRatio: number;
-  isIncludeBacklinks: boolean;
+  isShowLinkDirectionArrows: boolean;
```
Comment (treymo) [inline src/index.ts:25]: do you mind renaming this `shouldShowLinkDirection`?

File: src/index.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ interface GraphData {
   currentNoteID: string;
   nodeFontSize: number;
   nodeDistanceRatio: number;
-  isIncludeBacklinks: boolean;
```
Comment (treymo) [inline src/index.ts:25]: I don't think you intended to remove this feature. also can we call is `shouldIncludeBacklinks`

File: src/index.ts [hunk 3]
```diff
@@ -186,6 +186,9 @@ async function fetchData() {
   const isIncludeBacklinks = await joplin.settings.value(
```
Comment (treymo) [inline src/index.ts:186]: Can we call is `shouldIncludeBacklinks`?

File: src/index.ts (hunk 1 above)
Comment (goliath-walker) [reply src/index.ts:25]: Good point you catch here. Consistent naming convention is ... hard :-). 
`shouldXXX` sounds like you express a preference, and the system will decide ultimately. Instead my suggestion would be either 
a) to always prepend `is` for boolean attributes - `isIncludeBacklinks`, `isShowLinkDirectionArrows` or
b) simply use more fluent language - `includeBacklinks`, `showLinkDirectionArrows`.

I am in favor of b). What do you think?

File: src/index.ts (hunk 3 above)
Comment (goliath-walker) [reply src/index.ts:186]: See https://github.com/treymo/joplin-link-graph/pull/55#discussion_r811449353

File: src/index.ts (hunk 1 above)
Comment (treymo) [reply src/index.ts:25]: Good points, let's go with b.  I liked the prefix so it was easy to know it's a bool, but now we have TypeScript annotations for that :)

**Review by treymo — APPROVED** [review]

**Pull request merged**